### ISSUE TITLE
Review site links to ensure they are descriptive

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3088]
Links are present that are not descriptive of the destination or purpose in or out of context. “I located duplicate ‘a’ links which are confusing as I do not know what their purpose is.” Ensure that links contain text that is unique and descriptive to enable users to accurately determine their destination or purpose (`aria-label=”___”`).

This one is for the whole site - we need to look for any overly-generic links which wouldn't make sense out of context.

Update: High Contrast link was the main one, solved with https://github.com/ONSdigital/sdg-indicators/issues/3087, but ask DAC for any others. The survey link 'click here' also needs changing.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [ ] Windows: IE11
- [ ] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Review site links to ensure they are descriptive](https://trello.com/c/141gxEYQ/612-review-site-links-to-ensure-they-are-descriptive)

**Comment by Norric1Admin** [comment]
Look through 'elements list' for other nondescriptive links (e.g. 'read more')
JAWs insert + F3   NVIDIA insert + F7 (NVIDIA is free)

**Comment by Norric1Admin** [comment]
The links '@ons' and 'FAQ' are read weirdly by the Samsung SR ("at ouhns", "Fay-qwee", making them confusing. Change 'FAQ' to 'Frequently asked questions' and preface Twitter link with 'Twitter: ',

**Comment by Norric1Admin** [comment]
@samuelwillis8 to review a cross-section of indicators (2-3) for each goal, and all the non-stat ones, looking for any links which are really unclear out of context (e.g. "see here"). Don't screen-read everything, but if there's an acronym/initialism please give it the once-over to see if the screenreader mangles it.

**Comment by samuelwillis8** [comment]
**hello**

**Comment by Norric1Admin** [comment]
Thanks @samuelwillis8! Really helpful.

**Comment by Norric1Admin** [comment]
@Norric1Admin to complete this.

**Comment by Norric1Admin** [comment]
Branch missing (404) - I have rebuilt it and will continue when it's back.
Update: Works correctly on Firefox and Edge. It's also working fine on Samsung Chrome and Internet. However, the links '@ons' and 'FAQ' are read weirdly by the Samsung SR ("at ouhns", "Fay-qwee", making them confusing. @SavvasStephanides let's i) change 'FAQ' to 'Frequently asked questions' and ii) preface Twitter link with 'Twitter: ', as part of 

TO DO

**Comment by Norric1Admin** [comment]
@Norric1Admin Remember to add the upcoming metadata updates to `DAC-TEST`!

**Comment by Norric1Admin** [comment]
@brockfanning @LucyGwilliamAdmin 
Rebuilt both open-sdg and sdg-indicator branches from scratch. This is still failing.
https://circleci.com/gh/open-sdg/open-sdg/466?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
https://travis-ci.org/ONSdigital/sdg-indicators/builds/551791528?utm_source=github_status&utm_medium=notification

**Comment by AnnCorp** [comment]
Expert review has improved site content. DAC testing of improved content scheduled for August.
